elide-core/src/main/java/com/yahoo/elide/Elide.java

```diff
@@ -125,7 +125,7 @@ public Elide(AuditLogger auditLogger, DataStore dataStore, EntityDictionary dict
                 ActivePermissionExecutor::new,
                 Collections.singletonList(new DefaultFilterDialect(dictionary)),
                 Collections.singletonList(new DefaultFilterDialect(dictionary)),
-                false
+                true
```

Comment:
We can just remove this option.  It was temporary for Elide 2.0 to allow for FilterExpressions.